packages/generator/src/edmx-to-vdm/v2/function-import.ts

```diff
@@ -29,7 +30,11 @@ export function generateFunctionImportsV2(
         formatter
       ),
       httpMethod,
-      returnType: parseReturnTypes(f.ReturnType, entities, complexTypes)
+      returnType: parseFunctionImportReturnTypes(
```

Comment:
See my comment in `action-function-return-types.ts`. If the type is important here just add the `as` here.

Reply:
see above